docker/README.rst

```diff
@@ -60,6 +60,10 @@ If running on OS X, install then run the `Docker Desktop <https://www.docker.com
 
         Expect to wait a few minutes the first time you start the Docker Desktop. Don't proceed until the Docker Desktop's initialization is complete.
 
+        On OS X 12.4, Apple's new security features require you to give Docker `additional permissions <https://support.apple.com/guide/mac-help/control-access-to-files-and-folders-on-mac-mchld5a35146/mac>`_ if you place the files you install below in the Documents folder.
```

Comment:
We should recommend against installing anything in the Documents folder.  For most mac users this is a folder that is synced to iCloud and is not really an appropriate place for doing development work.

Reply:
I would have them create a subdirectory of their home directory called Runestone So that everything we installed gets installed under Runestone.

Reply:
Added -- I call the directory `rsdocker` to be a bit more descriptive.